CLOUD9_SETUP.md

```diff
@@ -25,7 +25,7 @@ Note: If you encounter any issues, try checking the Troubleshooting section at t
 - "Next Step" > "Configure settings" setting (can leave as defaults)
     ```
     [Environment type] Create a new EC2 instance for environment (direct access)
-    [Instance type] t2.micro (1 GiB RAM + 1 vCPU)
+    [Instance type] m4.4xlarge (64 GiB RAM + 16 vCPU)
```

Comment:
Is there a cheaper instance that can be used? (Note that m4.4xlarge is almost 100x more expensive than the previous one [[1]](https://tiny.amazon.com/wdntyn81))

Reply:
Smaller instantType will be out of disk or very slow with the latest AWS CDK and also new command we put in docker file which installed a lot of new dependencies. That might degrade the customer experiences. 
I think it is okay that we recommend the instance type that provide optimal customer experiences. They are free to choose cheaper instanceType if they want to compromise the experiences with cheaper instanceType.

Reply:
updated!